src/test/encoding/types.h

```diff
@@ -448,6 +448,11 @@ TYPE(rgw_obj)
 #include "rgw/rgw_log.h"
 TYPE(rgw_log_entry)
 
+#include "rgw/rgw_meta_sync_status.h"
+TYPE(rgw_meta_sync_info)
+TYPE(rgw_meta_sync_marker)
+TYPE(rgw_meta_sync_status)
+
```

Comment:
these types are already present just below under `rgw_sync.h`

Reply:
As those types are declared in rgw_meta_sync_status.h, it seems better to keep them here or remove the one from rgw_sync ?

Reply:
agreed!